tests/pos/byname-inline.scala

```diff
@@ -0,0 +1,39 @@
+final class ByName1(val x: Int) extends AnyVal:
+  inline def isEmpty: Boolean = x == 1
```

Comment:
I do not believe we have a way to make this work. The JVM would probably inline these calls anyway.